infrastructure/aat.tfvars

```diff
@@ -0,0 +1,4 @@
+infrastructure_env = "test"
+node_environment = "aat"
+tribunals_case_api = "http://sscs-tribunals-api-aat.service.core-compute-aat.internal/"
```

Comment:
i would build this dynamically in the `main.tf` file,
https://github.com/hmcts/cmc-claim-store/blob/cnp/infrastructure/main.tf#L60
it's not expected that you hard code the ase name